crates/pico_structures/src/schematic.rs

```diff
@@ -25,12 +25,22 @@ pub enum SchematicError {
     AirNotFound,
 }
 
-#[derive(Default)]
+#[derive(Default, Clone)]
```

Comment:
Would be cool if we don't have to clone the schematic, it may end up using a lot of memory for large schematics.
Same for block entities below.